docs/dev/tutorials/README.md

```diff
@@ -4,7 +4,7 @@ Learn how to build dApps on the zkSync protocol with our how-to guides
 
 ## Table of contents
 
-- [Hello world](../developer-guides/hello-world.md)
+- [Quickstart tutorial](../developer-guides/hello-world.md)
```

Comment:
And here, just "Quickstart", should be fine.

Reply:
fixed